extension/popup/popupView.jsx

```diff
@@ -578,7 +578,12 @@ this.popupView = (function() {
           ) : null}
         </div>
         {next ? (
-          <div id="search-show-next">
+          <a
+            id="search-show-next"
+            href={new URL(next.url)}
```

Comment:
Why `{new URL(...)}`? Couldn't this just be `next.url`?

But we should probably make this a bit more complex, and it should do the same thing that saying "next" does. Maybe this is fine:

```js
        browser.runtime.sendMessage({
          type: "runIntent",
          "next",
        });
```

At least that's a fine starting point.